src/compiler/scala/tools/nsc/settings/MutableSettings.scala

```diff
@@ -455,6 +455,7 @@ class MutableSettings(val errorFn: String => Unit)
     type T = Boolean
     protected var v: Boolean = false
     override def value: Boolean = v
+    def boolValue: Boolean = v
```

Comment:
There is an implicit conversion from BooleanSetting to Boolean that obviates a member. It currently resides in `reflect.internal.settings.MutableSettings`. As a matter of taste, `boolValue` is not my favorite name. :)